BGL/include/CGAL/boost/graph/copy_face_graph.h

```diff
@@ -208,8 +212,49 @@ void copy_face_graph(const SourceMesh& sm, TargetMesh& tm,
                        sm_vpm, tm_vpm);
 }
 
+template<typename PMAP>
+struct Output_iterator_functor
```

Comment:
the name is too generic

Reply:
So what ? It's a non documented implementation detail in the internal namespace